package.json

```diff
@@ -17,5 +17,12 @@
     "eslint": "^5.16.0",
     "prettier": "https://github.com/Automattic/wp-prettier/releases/download/wp-1.16.4/wp-prettier-1.16.4.tgz",
     "webpack": "^4.29.6"
+  },
+  "scripts": {
```

Comment:
There are two `scripts` objects in `package.json`. The first one looks to be left over from before—should be removed.

Reply:
This seems a bit confusing, but I believe there's no actual double `scripts` object, not anymore at least.

Clicking at the `package.json` header text of the file content snippet we're looking at right here in this conversation just above your comment, it opens a specific commit which shows double `scripts` objects. But looking at [this branch's (latest) `package.json` file](https://github.com/Automattic/newspack-content-converter/blob/add/release-build-script/package.json) doesn't show double `scripts`, same as my local file when I pull from this branch.